src/Microsoft.AspNetCore.Html.Abstractions/HtmlEncodedString.cs

```diff
@@ -17,17 +17,20 @@ public class HtmlEncodedString : IHtmlContent
         /// </summary>
         public static readonly IHtmlContent NewLine = new HtmlEncodedString(Environment.NewLine);
 
-        private readonly string _value;
-
         /// <summary>
         /// Creates a new <see cref="HtmlEncodedString"/>.
         /// </summary>
         /// <param name="value">The HTML encoded value.</param>
         public HtmlEncodedString(string value)
         {
-            _value = value;
+            Value = value;
         }
 
+        /// <summary>
+        /// Gets the HTML encoded value.
+        /// </summary>
+        public string Value { get; }
```

Comment:
Why? And, if this property is useful, does it need to be on `IHtmlContent` or one of the other `interface`s?